content/_changelogs/10.3.0.md

```diff
@@ -23,8 +23,8 @@ _Released 6/28/2022_
   fixed various issues with cookies when testing across multiple origins. Fixed
   [#21363](https://github.com/cypress-io/cypress/issues/21363) and
   [#20685](https://github.com/cypress-io/cypress/issues/20685).
-- Updated Cypress to use posix paths to fix an issue on
-  Windows when a project required `node-ts`. Fixes
+- Updated Cypress to use posix paths to fix an issue on Windows when a project
+  required `node-ts`. Fixes
```

Comment:
This is an unrelated change but the markdown linter was failing and `prettier` made this change when I ran `yarn fix:markdown`.